test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/IterableTest.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.lang
+
+import java.lang.{Iterable => JIterable}
+import java.util
```

Comment:
Prefer using
```scala
import java.{util => ju}
```
then `ju.Whatever` of you need this. Importing `util` is confusing, because `scala.util` also exists.